src/main/java/slimeknights/tconstruct/plugin/crt/managers/PartBuilderManager.java

```diff
@@ -0,0 +1,43 @@
+package slimeknights.tconstruct.plugin.crt.managers;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker.api.item.IItemStack;
+import com.blamejared.crafttweaker.api.managers.IRecipeManager;
+import com.blamejared.crafttweaker.impl.actions.recipes.ActionAddRecipe;
+import com.blamejared.crafttweaker.impl_native.item.ExpandItem;
+import net.minecraft.item.Item;
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraft.util.ResourceLocation;
+import org.openzen.zencode.java.ZenCodeType;
+import slimeknights.tconstruct.library.recipe.RecipeTypes;
+import slimeknights.tconstruct.library.recipe.partbuilder.PartRecipe;
+import slimeknights.tconstruct.library.tinkering.IMaterialItem;
+
+@ZenRegister
+@ZenCodeType.Name("mods.tconstruct.Partbuilder")
+public class PartBuilderManager implements IRecipeManager {
+  
+  @ZenCodeType.Method
+  public void addRecipe(String name, String pattern, int cost, Item output, int outputCount) {
```

Comment:
Call this something like `addMaterialRecipe`, I want to add support to the part builder for non-material recipes, sorta like the stonecutter.